src/codegen/browser/test.ts

```diff
@@ -130,14 +134,20 @@ function buildBrowserNodeGraph(events: BrowserEvent[]) {
         }
 
       case 'click': {
```

Comment:
I was thinking of going for a more general approach of detecting whether two things happened in parallel and wrap things in `Promise.all` if that was the case.

But this is MUCH simpler and I like that. 😬